backend/Makefile

```diff
@@ -97,3 +97,10 @@ db/tunnel:
 	ssh -f -T -N \
 		-o ServerAliveInterval=${SSH_SERVER_ALIVE_INTERVAL_IN_SECONDS} -o ServerAliveCountMax=${SSH_SERVER_ALIVE_COUNT_MAX} \
 		-L 5432:${endpoint}:5432 bastion.${DEPLOYMENT_STAGE}.single-cell.czi.technology
+
+db/download-prod:
```

Comment:
still need this?

Reply:
I was considering leaving it in as a convenience method to grab the prod db locally. What do you think?